### PR TITLE
Resolved the port conflicts

### PR DESCRIPTION
**Pull request by Gayanthakaushal**

### CODE REVIEW

File: pattern-1/artifacts/apim-analytics/wso2apim-analytics-1-deployment.yaml [hunk 1]
```diff
@@ -35,11 +35,7 @@ spec:
       - image: docker.wso2.com/wso2am-analytics-kubernetes:2.1.0
         name: wso2apim-analytics
         imagePullPolicy: IfNotPresent
-        resources:
-          limits:
-            memory: "4Gi"
-          requests:
-            memory: "4Gi"
```
Comment (imesh) [inline pattern-1/artifacts/apim-analytics/wso2apim-analytics-1-deployment.yaml]: Any reason to remove the resource limits?

File: pattern-1/artifacts/apim-analytics/wso2apim-analytics-2-deployment.yaml [hunk 2]
```diff
@@ -35,11 +35,7 @@ spec:
       - image: docker.wso2.com/wso2am-analytics-kubernetes:2.1.0
         name: wso2apim-analytics
         imagePullPolicy: IfNotPresent
-        resources:
-          limits:
-            memory: "4Gi"
-          requests:
-            memory: "4Gi"
```
Comment (imesh) [inline pattern-1/artifacts/apim-analytics/wso2apim-analytics-2-deployment.yaml]: Any reason to remove the resource limits?

Comment (Gayanthakaushal) [reply pattern-1/artifacts/apim-analytics/wso2apim-analytics-2-deployment.yaml]: Nothing special reason. But in our machine which we use to deploy the artifacts not have that much of memory, it causes to be a container create an issue.

Comment (imesh) [reply pattern-1/artifacts/apim-analytics/wso2apim-analytics-2-deployment.yaml]: A good point! May be we can fix that in a different pull request. Shall we add it back and merge this?

**Pull request merged**